FashionConnect_api/email.js

```diff
@@ -0,0 +1,35 @@
+import nodemailer from "nodemailer";
```

Comment:
I think all this code is already in your previous PR. Can you update this PR for only the frontend changes?